README.md

```diff
@@ -261,4 +261,4 @@ Figure A1 is generated by the regression estimation step (`codes/models/alt_grow
 #### Appendix Figure A2
 
 1. `Rscript codes/data/korea/make_JHU_comparison_data.R`: Creates [data/interim/korea/KOR_JHU_data_comparison.csv](data/interim/korea/KOR_JHU_data_comparison.csv)
-2. `python codes/plotting/figA2.py`: Generates 2 outputs that constitute Figure A2 (`results/figures/appendix/figA2-1.pdf` and `results/figures/appendix/figA2-2.pdf`)
```

Comment:
I changed all figures and tables to go in a results folder. Will be helpful if we make this repo a [Code Ocean](https://codeocean.com/) capsule which i think would be cool and very little marginal extra work once the repo is clean. Can you undo this change?